MyString.java

```diff
@@ -18,13 +18,40 @@ public static void main(String[] args) {
 
     /** Returns the lowercase version of the given string. */
     public static String lowerCase(String str) {
-        // Replace the following statement with your code
-        return null;
+       String lowered = "";
+        for (int i = 0; i < str.length(); i++) {
+            if(str.charAt(i) >= 97 && str.charAt(i) <= 122){
+                lowered += str.charAt(i);
+            } else if(str.charAt(i) >= 65 && str.charAt(i) <= 90){
+                lowered +=(char) ((int) str.charAt(i)+ 32);
+            } else if (str.charAt(i) == ' ' ||(str.charAt(i) >= 48 && str.charAt(i) <= 57)){
+                lowered += str.charAt(i);
+            }
+        }
+        return lowered;
     }
 
     /** If str1 contains str2, returns true; otherwise returns false. */
     public static boolean contains(String str1, String str2) {
-        // Replace the following statement with your code
-        return false;
+        if (str2.length() > str1.length()) return false; 
+        if (str2.isEmpty()) return true; 
```

Comment:
Acceptable, but see the other 2 options I mentioned in previous comments 
with using tools from lectures/recitations